app/controllers/api/v2_alpha/notifications_controller.rb

```diff
@@ -125,11 +125,11 @@ def pagination_collection
   end
 
   def browserable_params
-    params.permit(:include_filtered, types: [], exclude_types: [])
+    params.permit(:include_filtered, types: [], exclude_types: [], grouped_types: [])
```

Comment:
Sorry, I know this is not new with this PR, but I only just noticed that all other places where `params` are permitted perform a `#slice` before the `#permit`. Is there a reason why this is omitted here?

Reply:
I believe this suppresses the warning (or even exception if configured) if unpermitted parameters are present.